### ISSUE TITLE
[FEATURE] Support BLOOMZ

### ISSUE THREAD

**Issue by zhanyuanucb** [alpa-projects/alpa#815]
**System information**
- Alpa version: v0.2.2
- Are you willing to contribute it (Yes/No): Yes

**Describe the new feature and the current behavior/state**
BLOOMZ model family is available [here](https://huggingface.co/bigscience/bloomz). It uses the same model architect as BLOOM's, so it is easy to extend the `llm_serving` example to serve BLOOMZ.

**Will this change the current API? How?**
At least need to change these lines
https://github.com/alpa-projects/alpa/blob/fcd560d58e680b6d3c5098504242b49f527549ee/examples/llm_serving/model/bloom_model.py#L536-L573

I've tested `bloomz-560m` to `bloomz-7b1` in my fork:

https://github.com/zhanyuanucb/alpa/blob/e196638768392d22d55af41ada6f85b07abe69c4/examples/llm_serving/model/bloom_model.py#L536-L557

**Describe alternatives you've considered**

**Additional context**

**Comment by zhuohan123** [comment]
Hi! Please feel free to submit a PR for your proposed change.

**Comment by zhanyuanucb** [comment]
I haven't tested the full-size BLOOMZ model. I can do it later. But if anyone can test it out, feel free to post what they find here.

**Comment by zhisbug** [comment]
You can submit a PR from your fork?

**Comment by zhanyuanucb** [comment]
@zhuohan123 @zhisbug 
I found that the deployment is not stable. Sometimes it may fail and get stuck after downloading the model weights.
I need to look into this.

**Comment by zhanyuanucb** [comment]
@zhuohan123 @zhisbug 
Seems like the deployment failure only happens for `bloomz-7b1`, and it is due to an unexpected exit in this function:
https://github.com/alpa-projects/alpa/blob/b56e84319667b2ce31443ec9757cd7f0ae41d654/examples/llm_serving/model/wrapper.py#L586-L618

More specifically, between line 600 and line 605, because I saw `Load the pre-trained pytorch weights of` printed out in the log, but `Convert the weights to alpa format under` was not printed out. I couldn't find other error messages in the logs, and the resources should be enough since I could serve `bloomz-3b` and `bloom-7b1`.

If I run the `download_weights("bloomz-7b1", "/models/bloomz-7b1-np")` separately, the weight conversion can be done successfully. Once I got the converted weights, the model serving had no problem.

Does this have anything to do with some timeout in Ray? It would be helpful if someone can point me to some related part of the code.
